### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andriykooo**
[DEMO LINK](https://Andriykooo.github.io/react_sum/)

### CODE REVIEW

**Review by Dima-Batiuk — APPROVED** [review]
great!

**Comment by Dima-Batiuk** [comment]
I think you nedd to add whitespaces between numbers)

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```
Comment (aliceblunt8) [inline src/components/Sum.jsx:17]:
```suggestion
  {`Sum of ${a} and ${b} is ${a + b}`}
```

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
 
 const App = () => (
-  <h1>React sum</h1>
+  <div>
+    <h1>React sum</h1>
+
+    <Sum
+      a={2}
+      b={2}
+    />
```
Comment (aliceblunt8) [inline src/App.js:12]:
```suggestion
    <Sum a={2} b={2} />
```

**Review by aliceblunt8 — CHANGES_REQUESTED** [review]

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**